### PR TITLE
Add the subtext to the list of parameters in the conversational subentities request

### PR DESCRIPTION
**Pull request by debajyoti251088**
We received feedback that certain apps want to use a subtext that will be displayed in the deeplink chiclet for the conversation. Adding that parameter in here.

### CODE REVIEW

**Comment by sachingupta** [comment]
merge latest master to your feature branch.

**Review by sachingupta — DISMISSED** [review]

**Comment by debajyoti251088** [comment]
> merge latest master to your feature branch.

File: src/public/interfaces.ts [hunk 1]
```diff
@@ -418,6 +418,11 @@ export interface OpenConversationRequest {
    */
   entityId: string;
 
+  /**
+   * The subtext of the tab
```
Comment (ydogandjiev) [inline src/public/interfaces.ts]: Can we provide a better description here? I think this should say something like "Description of the item being discussed" or something along those lines.

File: src/public/interfaces.ts [hunk 2]
```diff
@@ -418,6 +418,11 @@ export interface OpenConversationRequest {
...
+   */
+  subtext?: string;
```
Comment (ydogandjiev) [inline src/public/interfaces.ts]: How did we come up with the `subtext` name? I think it might be better to call this `subtitle` or `description`.

Comment (debajyoti251088) [reply src/public/interfaces.ts]: Loki provided that in the work item. Will sync with him.

**Comment by TrevorJoelHarris** [comment]
This PR is stale and will be closed, please feel free to re-open it if your intention is still to check into main.

**Pull request closed without merging**